// An error is randomly returned to demonstrate how the error is propagated
// to the 'onError' function when an error occurs in the 'onTrigger'
// function. Therefore position of the error might differ and can be appeared in random order.
# To run this sample, navigate to the directory that contains the
# `.bal` file and issue the `ballerina run` command.
$ ballerina run task_timer.bal
Timer task demo
Cleaning up...
1
Cleaning up...
2
Cleaning up...
3
Cleaning up...
4
Cleaning up...
5
[ERROR] cleanup failed: Cleanup error {}
Cleaning up...
6
Cleaning up...
7
Cleaning up...
8
Cleaning up...
9
Cleaning up...
10
Stopped timer
